.github/workflows/scheduled_sweep.yml

```diff
@@ -0,0 +1,19 @@
+---
+on:
+  schedule:
+    - cron: '14 7 15 * *'
+name: Scheduled sweep
+jobs:
+  clippy:
+    name: Sweep
+    runs-on: [self-hosted, heavy]
+    steps:
+      - name: Clone the repository
+        uses: actions/checkout@v2
+      - name: Install cargo-make
+        uses: actions-rs/cargo@v1
+        with:
+          command: install
+          args: --debug cargo-make
+      - name: Run sweep
+        run: cargo make sweep
```

Comment:
Could you describe what exactly sweep does? Which files are affected?

Reply:
This sweep simply cleans builds that are older than 30 days.

Reply:
Please name files location :) Because there's some chance that this workflow doesn't do what you intended

Reply:
What ever is in the target build folder. If that gets cleaned up each time anyways, then this is unnecessary.

Reply:
Yep, it gets cleaned up anyways)